charts/alfresco-connector-ms365/tests/deployment-connector-ms365_test.yaml

```
@@ -44,8 +44,18 @@ tests:
         test_data: 80
     asserts:
         - equal:
-            path: spec.template.spec.containers[0].env[0]
+            path: spec.template.spec.containers[0].env[2]
             value:
                 name: test_data
                 value: "80"
           template: deployment-connector-ms365.yaml
+
+  - it: should set environment variables correctly with quotes
+    values: *testvalues
+    asserts:
+        - equal:
+            path: spec.template.spec.containers[0].env[2]
+            value:
+                name: JAVA_OPTS
+                value: "-Dalfresco.base-url=http://$(REPO_HOST):$(REPO_PORT)"
```

please set some java_opts to test they are not lost.
e.g:
```suggestion
    set:
      environment:
        JAVA_OPTS: "-Xmx 2GB"
    asserts:
        - equal:
            path: spec.template.spec.containers[0].env[2]
            value:
                name: JAVA_OPTS
                value: "-Xmx 2GB -Dalfresco.base-url=http://$(REPO_HOST):$(REPO_PORT)"
``